src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -686,6 +688,7 @@ public static void removeUnfinishedCompactionLeftovers(CFMetaData metadata, Map<
 
```

Comment:
Let's bump the `logger.trace("Unfinished compactions of {}.{} reference missing sstables of generations {}"` to at least debug?